jdaviz/configs/specviz/helper.py

```diff
@@ -128,18 +128,23 @@ def get_spectra(self, data_label=None, subset_to_apply=None, apply_slider_redshi
 
             return output_spectra
 
-    def get_spectral_regions(self):
+    def get_spectral_regions(self, use_display_units=False):
         """
         A simple wrapper around the app-level call to retrieve only spectral
         subsets, which are now returned as SpectralRegions by default.
 
+        Parameters
+        ----------
+        use_display_units: bool, optional
+            Whether to convert to the display units defined in the <unit-conversion> plugin.
```

Comment:
Unless the carrots are doing some special Sphinx formatting, we can get rid of them.
```suggestion
            Whether to convert to the display units defined in the Unit Conversion plugin.
```

Reply:
ah, I was hoping it would link to the ref for the plugin docs, but[ apparently does not](https://jdaviz--2132.org.readthedocs.build/en/2132/api/jdaviz.configs.specviz.helper.Specviz.html#jdaviz.configs.specviz.helper.Specviz.get_spectral_regions).

Reply:
What about " :ref:`Unit Conversion <unit-conversion>` "?

Reply:
ah of course (other projects I work on handle without the ref and I get confused switching).  Updated now to use the actual link, thanks @javerbukh!